package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@team-griffin/uteals",
+  "version": "1.0.0",
+  "description": "Utilities for Tealium",
+  "main": "dist/cjs/uteals.js",
+  "module": "dist/es/uteals.js",
+  "repository": "git@github.com:team-griffin/uteals.git",
+  "author": "Jack Ellis <jack.ellis@godaddy.com>",
+  "license": "MIT",
+  "scripts": {
+    "prepublish": "npm run build",
+    "transpile": "babel src -d lib --copy-files",
+    "lint": "npm run lint:js && npm run flow",
+    "lint:js": "eslint -c ./config/.eslintrc.js 'src/**/*.js'",
+    "lint:js:fix": "eslint --fix -c ./config/.eslintrc.js 'src/**/*.js'",
+    "flow": "flow check; test $? -eq 0 -o $? -eq 2",
+    "build:dev": "cross-env NODE_ENV=development rollup --config ./rollup/development.js",
+    "build:prod": "cross-env NODE_ENV=production rollup --config ./rollup/production.js",
+    "build:entry": "cross-env NODE_ENV=production rollup --config ./rollup/entry.js",
+    "build": "yarn build:dev && yarn build:prod && yarn build:entry",
+    "test": "cross-env NODE_ENV=test nyc mocha -- \"src/**/*.test.js\"",
+    "postmerge": "yarn"
+  },
+  "devDependencies": {
+    "@team-griffin/eslint-config": "^3.0.3",
+    "babel-cli": "^6.23.0",
+    "babel-core": "^6.1.21",
+    "babel-eslint": "^8.0.3",
+    "babel-plugin-external-helpers": "^6.22.0",
+    "babel-plugin-ramda": "^1.4.3",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-flow": "^6.23.0",
+    "chai": "^4.0.2",
+    "cross-env": "^5.0.1",
+    "eslint": "^4.13.0",
+    "eslint-plugin-flowtype": "^2.30.4",
+    "flow-bin": "^0.72.0",
+    "mocha": "^4.0.1",
+    "nyc": "^11.0.2",
+    "rollup": "^0.54.0",
+    "rollup-plugin-babel": "^3.0.3",
+    "rollup-plugin-local-resolve": "^1.0.7",
+    "rollup-plugin-memory": "^3.0.0",
+    "rollup-plugin-replace": "^2.0.0",
+    "rollup-plugin-uglify-es": "^0.0.1",
+    "sinon": "^5.0.7"
+  },
+  "nyc": {
+    "require": [
+      "babel-register"
+    ],
+    "sourceMap": false,
+    "instrument": false,
+    "include": [
+      "src/**/*.js"
+    ]
+  },
+  "dependencies": {
+    "browser-cookies": "^1.2.0",
```

Comment:
I would say that `browser-cookies` is an optional dependency. I'll let this through as optionals get odd behaviour and determinism.